### PR TITLE
add senario: rename collection in a new db

### PR DESCRIPTION
**Pull request by NicoYuan1986**
link: #27385

### CODE REVIEW

File: tests/python_client/testcases/test_database.py [hunk 1]
```diff
@@ -523,6 +523,95 @@ def test_using_db_not_existed(self):
...
+        c_name = "collection_same"
+        self.init_collection_wrap(c_name)
+        assert self.utility_wrap.list_collections()[0] == [c_name]
+
+        # create a new database
+        db_name = cf.gen_unique_str("db")
+        self.database_wrap.create_database(db_name)
+        self.database_wrap.using_database(db_name)
+
+        # create a collection in new db using same name
```
Comment (yanliang567) [inline tests/python_client/testcases/test_database.py:547]: i sugesst assert there is no collection in the new db before create one.

**Review by yanliang567 — APPROVED** [review]

File: tests/python_client/testcases/test_database.py (hunk 1 above)
Comment (NicoYuan1986) [reply tests/python_client/testcases/test_database.py:547]: added. https://github.com/milvus-io/milvus/pull/28280/files#diff-e5be884a8489fffa6cff00449bff72d8a5e087ab39b64cc3acb45e096d0bfffc

**Pull request merged**